app/routes/bdb/components/BdbDetail.js

```diff
@@ -275,6 +292,10 @@ export default class BdbDetail extends Component<Props, State> {
                 company.studentContact.fullName) ||
                 '-'}`}
               meta="Studentkontakt"
+              link={`${String(
```

Comment:
Maybe create a util function for this? Gets a bit messy with template literals, String()'s and the && shortcutting to join the strings.